packages/tunnel/src/commands/deploy/index.ts

```diff
@@ -0,0 +1,86 @@
+import { existsSync } from 'node:fs';
+import path from 'node:path';
+
+import { isValidUrl } from '@logto/core-kit';
+import chalk from 'chalk';
+import type { CommandModule } from 'yargs';
+
+import { consoleLog } from '../../utils.js';
+
+import { type DeployCommandArgs } from './types.js';
+import { deployToLogtoCloud } from './utils.js';
+
+const tunnel: CommandModule<unknown, DeployCommandArgs> = {
+  command: ['deploy'],
+  describe: 'Deploy your custom UI assets to Logto Cloud',
+  builder: (yargs) =>
+    yargs.options({
+      auth: {
+        describe: 'Auth credentials of your Logto M2M application. E.g.: <app-id>:<app-secret>',
```

Comment:
will it be better to indicate this M2M app should  have Management API access?

Reply:
I think it would be too much for cli description maybe? I will definitely have this covered in docs, though

Reply:
or give a link to the docs should be fine, too

Reply:
Doc link added